src/webnn_native/ops/LeakyReluBase.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WEBNN_NATIVE_OPS_LEAKYRELUBASE_H_
+#define WEBNN_NATIVE_OPS_LEAKYRELUBASE_H_
+
+#include "webnn_native/ops/ClampBase.h"
+
+namespace webnn_native { namespace op {
+
+    class LeakyReluBase {
+      public:
+        LeakyReluBase(LeakyReluOptions const* options) {
+            mAlpha = options == nullptr ? 0.01 : options->alpha;
+        }
+        ~LeakyReluBase() = default;
+
+        const float* GetAlpha() const {
```

Comment:
why not get the float value directly? Getting a float pointer seems unnecessary.

Reply:
Yes, I forget to update this after refactoring codes. Thanks!